src/test/java/com/amazon/ion/impl/macro/MacroCompilerTest.kt

```diff
@@ -115,7 +115,7 @@ class MacroCompilerTest {
                 BoolValue(emptyList(), false),
             )
         ),
-        """(macro using_expr_group () (values (; 42 "hello" false)))""" shouldCompileTo TemplateMacro(
+        """(macro using_expr_group () (.values (.. 42 "hello" false)))""" shouldCompileTo TemplateMacro(
```

Comment:
Are we allowing `(..42 "hello" false)` (i.e., no whitespace after `..`)?

Reply:
In this case, they are already syntactically identical because of the rules for parsing symbols inside of s-expressions. For example, `(+x)` is equivalent to `( + x )`, and this falls under those same rules.

Reply:
Agreed, just wanted to make sure we're not planning to go out of our way to prohibit that in this case. I'm happy not to.